<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceData</class>
 <widget class="QWidget" name="SourceData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Source Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionImport">
   <property name="text">
    <string>Import Data</string>
   </property>
   <property name="toolTip">
    <string>import source data</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="toolTip">
    <string>clear all data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
